hw/top_earlgrey/data/ip/chip_clkmgr_testplan.hjson

```diff
@@ -0,0 +1,330 @@
+        "CLKMGR.ENABLE.IO_DIV4",
+        "CLKMGR.ENABLE.USB",
+      ]
+      tests: ["chip_sw_clkmgr_off_peri"]
+      bazel: ["//sw/device/tests:clkmgr_off_peri_test_fpga_cw310_test_rom"]
+    }
+    {
+      name: chip_sw_clkmgr_div
+      desc: '''Verify clk division logic is working correctly.
+
+            The IP level checks the divided clocks via SVA, and these are also bound at chip level.
+            Connectivity tests check peripherals are connected to the clock they expect.
+            Use the clkmgr count measurement feature to verify clock division.
+
+	    SiVal: These tests should be run after clock calibration is completed.
+	    Any LC cycle stage after calibration is done.
+	    Testing the software enabled external clock cycle measurements seems mute after clocks are calibrated.
+            '''
+      stage: V2
+      si_stage: SV1
+      features: [
+        "CLKMGR.MEAS_CTRL.REGWEN",
+        "CLKMGR.MEAS_CTRL.IO",
+        "CLKMGR.MEAS_CTRL.IO_DIV2",
+        "CLKMGR.MEAS_CTRL.IO_DIV4",
+        "CLKMGR.MEAS_CTRL.MAIN",
+        "CLKMGR.MEAS_CTRL.USB",
+      ]
+      tests: ["chip_sw_ast_clk_outputs",
+              "chip_sw_clkmgr_external_clk_src_for_sw_fast_test_unlocked0",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_test_unlocked0",
+              "chip_sw_clkmgr_external_clk_src_for_sw_fast_dev",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_dev",
+              "chip_sw_clkmgr_external_clk_src_for_sw_fast_rma",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_rma",
+              "manuf_cp_unlock_raw",
+	      "manuf_cp_yield_test"]
+      bazel: [
+        "//sw/device/tests:ast_clk_outs_test_fpga_cw310_test_rom",
+	"//sw/device/tests:clkmgr_external_clk_src_for_sw_fast_test_fpga_cw310_test_rom",
+	"//sw/device/tests:clkmgr_external_clk_src_for_sw_slow_test_fpga_cw310_test_rom",
+      ]
+    }
+    {
+      name: chip_sw_clkmgr_external_clk_src_for_lc
+      desc: '''Verify the clkmgr requests ext clk src during certain LC states.
+
+            On POR lc asserts lc_clk_byp_req on some LC states, and de-asserts
+            it when lc_program completes. This also triggers divided clocks to step down. It may be
+            best to verify this via SVA, unless we implement clock cycle counters.
+	    SiVal: There is no way to test clock frequencies before clocks are calibrated.
+	    So the only way to actually test this is by the success of OTP updates for calibration.
+	    This can be tested after the fact by clock measurements of the internal, calibrated clocks.
+            '''
+      stage: V2
+      si_stage: SV1
+      tests: [
+        "chip_sw_clkmgr_external_clk_src_for_lc",
+	"manuf_cp_unlock_raw",
+      ]
+    }
+    {
+      name: chip_sw_clkmgr_external_clk_src_for_sw
+      desc: '''Verify SW causes the clkmgr requests ext clk src during certain LC states.
+
+            In RMA and TEST_UNLOCKED lc states the external clock is enabled in response to
```

Comment:
From the spec, it includes DEV state too. [Clock Frequency Summary](https://opentitan.org/book/hw/ip/clkmgr/doc/theory_of_operation.html#clock-frequency-summary)

Reply:
Done.